externals/service_ittnotify.h

```diff
@@ -96,10 +96,19 @@ class ScopedTask
     #define DAAL_ITTNOTIFY_SCOPED_TASK(name)                                            \
         static daal::internal::ittnotify::StringHandle __ittnotify_stringhandle(#name); \
         daal::internal::ittnotify::ScopedTask __ittnotify_task(__ittnotify_domain, __ittnotify_stringhandle)
+
 #else
+    #include "service_profiler.h"
+
+    #undef DAAL_ITTNOTIFY_CONCAT
+    #define DAAL_ITTNOTIFY_CONCAT(x, y)  DAAL_ITTNOTIFY_CONCAT2(x, y)
```

Comment:
`undef` these defines `DAAL_ITTNOTIFY_CONCAT` and `__DAAL_UNIQUE_ID` after usage, e.g.,
```cpp
here <-------
#endif // __DAAL_ITTNOTIFY_ENABLE__
```